src/components/mediainfo/mediainfo.js

```diff
@@ -261,7 +261,13 @@ import 'emby-button';
 
                 minutes = minutes || 1;
 
-                miscInfo.push(`${Math.round(minutes)} mins`);
+                if (item.UserData?.PlaybackPositionTicks) {
+                    let remainingMinutes = (item.RunTimeTicks - item.UserData.PlaybackPositionTicks) / 600000000;
```

Comment:
It's not immediately obvious to new contributors why we're dividing by 600000000 so, for maintainability, it'd be good to decompose this operation a bit and add a comment about why and how.

Reply:
As a new contributor, I agree that it's not immediately obvious, so I just copied the magic number from a couple lines above.  I did a global search for RunTimeTicks and didn't really find it explained anywhere else, though it looks like it's 10,000 ticks per ms * 60,0000 ms per minute.  Do you think changing it to something like this would be helpful?

    // 10,000 ticks per millisecond * 60,000 milliseconds per minute = 600,000,000 ticks per minute
    const ticksPerMinute = 10000 * 60000;
    let remainingMinutes = (item.RunTimeTicks - item.UserData.PlaybackPositionTicks) / ticksPerMinute;